src/mopup/__main__.py

```diff
@@ -28,9 +29,24 @@
     default=False,
     help="don't actually download or install anything even if we're not up to date",
 )
-def main(interactive: bool, force: bool, minor: bool, dry_run: bool) -> None:
+@click.option(
+    "--uninstall",
+    default=None,
+    help="uninstall a specific Python version (e.g., 3.13)",
+    type=str,
+)
+def main(
+    interactive: bool, force: bool, minor: bool, dry_run: bool, uninstall: str | None
+) -> None:
     """MOPUp."""
-    libmain(interactive=interactive, force=force, minor_upgrade=minor, dry_run=dry_run)
+    if uninstall:
+        libuninstall(
+            version=uninstall, dry_run=dry_run, interactive=interactive, force=force
```

Comment:
The awkwardness of the command options here suggests to me that this should be a new subcommand. In particular it feels like we should be allowing the user to specify the version, since I would imagine a common requirement for this functionality would be "I installed 3.15 and I want to uninstall 3.13, but mopup is pointed at 3.15 now"  Previously, `--minor` would give you *some* level of control there, but this just ignores that option and doesn't add a new one.

Reply:
Alright, will move to a subcommand.